src/reducers/versions.tsx

```diff
@@ -424,29 +429,64 @@ export const createInternalVersion = (
   };
 };
 
-export const getVersionFiles = (versions: VersionsState, versionId: number) => {
-  return versions.versionFiles[versionId];
+type GetVersionEntriesKeyParams = {
+  versionId: number;
+  comparedToVersionId: number | null | undefined;
+};
+
+export const getVersionEntriesKey = ({
+  versionId,
+  comparedToVersionId,
+}: GetVersionEntriesKeyParams) => {
+  if (comparedToVersionId === undefined) {
```

Comment:
I suggest removing this if statement so that all keys are consistent.

For the `undefined` case, a key could look like:

```
vesionId=123;comparedToVersionId=undefined
```